OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/FullscreenDialogFragment.kt

```diff
@@ -141,14 +187,26 @@ class FullscreenLoaderDialogFragment : DialogFragment() {
         return spannable
     }
 
+    /**
+     * Method to get the remaining visible time for the loader
+     * As per requirements the loader needs to be visible for at least 3 seconds
+     */
+    fun getRemainingVisibleTime(): Long {
+        val totalVisibleTime = Calendar.getInstance().timeInMillis - loaderStartTime
+        return if (totalVisibleTime < MINIMUM_DISPLAY_DELAY)
+            MINIMUM_DISPLAY_DELAY - totalVisibleTime
+        else
+            0
+    }
+
     private fun resetPurchase() {
         iapViewModel.resetPurchase(false)
         dismiss()
     }
 
     companion object {
         const val TAG = "FULLSCREEN_LOADER"
-        const val FULLSCREEN_DISPLAY_DELAY: Long = 3_000
+        const val MINIMUM_DISPLAY_DELAY: Long = 3_000
```

Comment:
Why need to put the `_` in the number. if added by mistake, please remove it.

Reply:
we can use `_` in defining long values to increase the readability.